packages/features/portfolio/src/containers/portfolio-details/note-holdings-detail.tsx

```diff
@@ -0,0 +1,238 @@
+        </BackButtonContainer>
+        <HeaderContainer>
+          <TokenContainer>
+            <TokenIcon symbol={value.asset.symbol} size="xlarge" />
+            <Box
+              sx={{
+                display: 'flex',
+                flexDirection: 'column',
+                alignItems: 'flex-start',
+                justifyContent: 'flex-start',
+              }}
+            >
+              <Header variant="h1" sx={{ marginBottom: 0 }}>
+                {value?.asset.label}
+              </Header>
+              <Caption variant="caption">{value?.asset.caption}</Caption>
+            </Box>
+          </TokenContainer>
+          <Box
+            sx={{
+              display: 'flex',
+              flexDirection: 'column',
+              alignItems: 'flex-end',
+              justifyContent: 'flex-start',
+            }}
+          >
+            <Header variant="h2" sx={{ marginBottom: 0 }}>
+              {value.notePrice}
+            </Header>
+          </Box>
+        </HeaderContainer>
+
+        <ContentContainer>
+          <Title variant="h3">
+            <FormattedMessage
+              id="earningsSummary"
+              defaultMessage="Earnings Summary"
+            />
+          </Title>
+          <Row>
+            <Label>
+              <FormattedMessage id="notePrice" defaultMessage="Note Price" />
+            </Label>
+            <Column>
+              <Value>{value.notePrice}</Value>
+            </Column>
+          </Row>
+
+          <Row>
+            <Label>
+              <FormattedMessage id="totalNOTE" defaultMessage="Total NOTE" />
+            </Label>
+            <Column>
+              <Value>{value.totalNOTE.data?.[0]?.displayValue}</Value>
+              <Caption>{value.totalNOTE.data?.[1]?.displayValue}</Caption>
+            </Column>
+          </Row>
+        </ContentContainer>
+
+        <ContentContainer>
+          <Title variant="h3">
+            <FormattedMessage id="details" defaultMessage="Details" />
+          </Title>
+          {value.actionRow.subRowData.map((row, index) => (
+            <Row key={index}>
+              <Label>{row.label}</Label>
+              <Column>
+                <Value>{row.value}</Value>
+              </Column>
+            </Row>
+          ))}
+        </ContentContainer>
+      </Container>
+      {/* <ActionButtonContainer>
+        {value.actionRow.buttonBarData.map((button, index) => (
+          <Button key={index} variant="outlined" color="primary" fullWidth>
+            {button.buttonText}
+          </Button>
+        ))}
+      </ActionButtonContainer> */}
+    </>
+  );
+};
+
+const Container = styled(Box)({
+  display: 'flex',
+  flexDirection: 'column',
+  gap: 16,
+});
+
+const BackButtonContainer = styled(Box)(({ theme }) => ({
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'flex-start',
+  gap: theme.spacing(1),
+  padding: theme.spacing(1),
+}));
+
+const BackButton = styled(Typography)(({ theme }) => ({
```

Comment:
We should already have all matching typography components exported from `typography.tsx` the text sizes are labelled in Figma here. We should use those components instead to keep the app consistent. 

![image](https://github.com/user-attachments/assets/207f20b3-b274-4e62-ae19-9db1779ed6e4)